app/main/views/suppliers.py

```diff
@@ -202,50 +242,84 @@ def edit_supplier_case_study(casestudy_id):
     )
 
 
-def _validate_case_study(casestudy):
+def _validate_case_study(casestudy, extra_fields=[]):
```

Comment:
I just added https://github.com/AusDTO/dto-digitalmarketplace-utils/blob/master/react/response.py#L14 refactor to use this